test/test_readout_error.py

```diff
@@ -11,20 +11,20 @@
 # that they have been altered from the originals.
 
 """
-A Tester for the Readout Mitigation experiment
+A Tester for the Readout error experiment
 """
 
 
 import unittest
 from test.base import QiskitExperimentsTestCase
 import numpy as np
 from qiskit.quantum_info.operators.predicates import matrix_equal
-from qiskit_experiments.library import ReadoutMitigationExperiment
+from qiskit_experiments.library.characterization import LocalReadoutError, CorrelatedReadoutError
 from qiskit_experiments.framework import ExperimentData
 
 
 class TestMitigation(QiskitExperimentsTestCase):
-    """Test ReadoutMitigationExperiment"""
+    """Test Readout Error experiments"""
```

Comment:
Please add a test for parallel experiments. I'd like to see that they work because of the usage of the physical qubits inside `_run_analysis`, I think it's OK and that the composite experiment doesn't overwrite them, but it's good to verify.